ros-hsrc_description/install.yaml

```diff
@@ -0,0 +1,7 @@
+- type: target
+  name: hsr-setup
+
+- type: ros
+  source:
+    type: system
+    name: hsrc-description
```

Comment:
This target can be removed safely after removing it as a dependency in `hsrb_moveit_config` as it is just used in a launch file there.

Reply:
Although if you want to keep this target for any later use, I'm okay with it

Reply:
Keep it. I want to keep the changes to uostream limited.